src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -279,13 +279,25 @@ def fuzz(
       engine.FuzzResult
     """
 
-    def _filter_log(content):
-      """Filter unneeded content from log."""
-      result = ''
-      strip_regex = re.compile(r'^c\d+\s+\d+\s')
-      for line in content.splitlines():
-        result += strip_regex.sub('', line) + '\n'
-      return result
+    def _filter_log(content: str) -> str:
+      """Remove unnecessary prefix from each line of log.
+
+      e.g
+      [  565.723853] c4   8262 BUG: KASAN: ...
```

Comment:
Just curious: what does "c4  8262" mean?

Reply:
we are not entirely sure what it is. possibly pid. We do not need this prefix either way.